src/scene/light.js

```diff
@@ -637,6 +637,31 @@ class Light {
         return clone;
     }
 
+    /**
+     * Get conversion factor for luminance -> light specific light unit
+     * @param {number} type - The type of light
+     * @param {number} outerAngle - The outer angle of a spot light
+     * @param {number} innerAngle - The inner angle of a spot light
```

Comment:
Marking those params as optional
```suggestion
     * @param {number} [outerAngle] - The outer angle of a spot light
     * @param {number} [innerAngle] - The inner angle of a spot light
```